packages/react-ui/src/index.d.ts

```diff
@@ -35,6 +39,10 @@ export {
   TableWidgetUI,
   LegendWidgetUI,
   LEGEND_TYPES,
-  NoDataAlert
+  NoDataAlert,
+  // undocumented components
+  LegendCategories as _LegendCategories,
```

Comment:
@bbecquet, @Clebal just commented me about this: "components can't be exported with _" (in the past all _ exported elements were functions, indeed). Have you tried that works? If not, we can just keep your initial proposal, without "_"

Reply:
@VictorVelarde I've tested it locally and it's ok. I'm renaming them on import anyway (`import { _LegendCategories as LegendCategories } from '@carto/react-ui'`, so it's normal it works.
But I've also tested using them directly in JSX and that works too… Maybe that's weird, as React component names are supposed to start with an uppercase letter, but in the actual render tree they keep the name of component itself, not the exported alias.
![Capture d’écran de 2022-02-08 17-31-51](https://user-images.githubusercontent.com/243653/153032688-4452b8ee-2f67-491c-8cf3-8f346c3261ba.png)

So I would say there is no problem, especially because I'll be the only one to use them for now, but maybe by convention it's not a good practice to export them with `_`?

What's your take on that @Clebal?

Reply:
If "_" works, I think it's better to keep it for the moment, for consistency with other non-public api methods. I would go ahead with it, even if that forces to use an alias with importing (that also serves as a reminder to end users), but let's hear Sergio

Reply:
If an alias without "_" is used in the import, then the rule won't complain.
From my point of view: I wouldn't export it as internal, because in the near future, to accomplish some PS requests, I will document them and make them public. We can make them public now (avoiding a future breaking change in cloud-native), and document them before the publishing of the v1.2.

Reply:
Ok, I removed the _ and updated the changelog.
Thanks for your input @Clebal

Reply:
I agree with Victor approach. In my opinion, we should keep it with the underscore and not document the components yet.

Reply:
So, I've dropped the commit where I removed the `_`, they are back to being prefixed.
In any case I don't think that's such a big deal as it's so easy to change either way, let's move on with that :)

Reply:
Let's go for the moment with "_".